libkbfs/disk_block_cache_test.go

```diff
@@ -758,3 +760,51 @@ func TestDiskBlockCacheMoveBlock(t *testing.T) {
 	require.Equal(t, 1, cache.syncCache.numBlocks)
 	require.Equal(t, TriggeredPrefetch, prefetchStatus)
 }
+
+func TestDiskBlockCacheHomeDirPriorities(t *testing.T) {
+	t.Parallel()
+	t.Log("Test that blocks from a home directory aren't evicted when there" +
+		" are other options.")
+	cache, config := initDiskBlockCacheTest(t)
+	defer shutdownDiskBlockCacheTest(cache)
+
+	ctx := context.Background()
+
+	t.Log("Set home directories on the cache")
+	homeTLF := tlf.FakeID(11, tlf.Private)
+	cache.AddHomeTLF(ctx, homeTLF)
+	homePublicTLF := tlf.FakeID(11, tlf.Public)
+	cache.AddHomeTLF(ctx, homePublicTLF)
```

Comment:
Where do you seed blocks for these two TLFs?

Reply:
The helper that seeds blocks for tests seeds the first N TLF IDs, which include these.

Reply:
Hmm...not thrilled about how opaque that is.

Additionally, you can't have different TLFs have the same ID, even if the TLF type is different. The cache only indexes on TLF ID, not on type.
https://github.com/keybase/kbfs/blob/2ebbe681777cc1e03dbcc2ae84cefc6488290f3c/libkbfs/disk_block_cache.go#L418-L420
https://github.com/keybase/kbfs/blob/2ebbe681777cc1e03dbcc2ae84cefc6488290f3c/tlf/id.go#L175-L177